config_test.go

```diff
@@ -460,4 +464,67 @@ func (s *AdminSuite) TestConfigDefaults() {
 	s.Equal(defaultLogBufferingDuration, config.LoggerConfig.Buffer.DurationSeconds)
 	s.Equal("info", config.LoggerConfig.DefaultLevel)
 	s.Equal(defaultAmboyPoolSize, config.Amboy.PoolSizeLocal)
+	s.Equal("v1", config.Expansions["k1"])
+	s.Equal("v2", config.Expansions["k2"])
+}
+
+func (s *AdminSuite) TestKeyValPairsToMap() {
```

Comment:
should also test the nested functions

Reply:
Moved them to a different file and added more tests